ext/opentelemetry-ext-datadog/tests/test_datadog_exporter.py

```diff
@@ -72,26 +72,73 @@ def test_constructor_explicit(self):
         """Test the constructor passing all the options."""
         agent_url = "http://localhost:8126"
         exporter = datadog.DatadogSpanExporter(
-            agent_url=agent_url, service="explicit"
+            agent_url=agent_url, service="explicit",
         )
 
         self.assertEqual(exporter.agent_url, agent_url)
         self.assertEqual(exporter.service, "explicit")
-        self.assertIsNotNone(exporter.agent_writer)
+        self.assertIsNone(exporter.env)
+        self.assertIsNone(exporter.version)
+        self.assertEqual(exporter.tags, {})
+
+        exporter = datadog.DatadogSpanExporter(
+            agent_url=agent_url,
+            service="explicit",
+            env="test",
+            version="0.0.1",
+            tags="",
+        )
+
+        self.assertEqual(exporter.agent_url, agent_url)
+        self.assertEqual(exporter.service, "explicit")
+        self.assertEqual(exporter.env, "test")
+        self.assertEqual(exporter.version, "0.0.1")
+        self.assertEqual(exporter.tags, {})
+
+        exporter = datadog.DatadogSpanExporter(
+            agent_url=agent_url,
+            service="explicit",
+            env="test",
+            version="0.0.1",
+            tags="team:testers",
+        )
+
+        self.assertEqual(exporter.agent_url, agent_url)
+        self.assertEqual(exporter.service, "explicit")
+        self.assertEqual(exporter.env, "test")
+        self.assertEqual(exporter.version, "0.0.1")
+        self.assertEqual(exporter.tags, {"team": "testers"})
 
     @mock.patch.dict(
         "os.environ",
-        {"DD_TRACE_AGENT_URL": "http://agent:8126", "DD_SERVICE": "environ"},
+        {
+            "DD_TRACE_AGENT_URL": "http://agent:8126",
+            "DD_SERVICE": "test-service",
+            "DD_ENV": "test",
+            "DD_VERSION": "0.0.1",
+            "DD_TAGS": "team:testers",
+        },
     )
     def test_constructor_environ(self):
         exporter = datadog.DatadogSpanExporter()
 
         self.assertEqual(exporter.agent_url, "http://agent:8126")
-        self.assertEqual(exporter.service, "environ")
+        self.assertEqual(exporter.service, "test-service")
+        self.assertEqual(exporter.env, "test")
+        self.assertEqual(exporter.version, "0.0.1")
+        self.assertEqual(exporter.tags, {"team": "testers"})
         self.assertIsNotNone(exporter.agent_writer)
 
     # pylint: disable=too-many-locals
-    @mock.patch.dict("os.environ", {"DD_SERVICE": "test-service"})
+    @mock.patch.dict(
+        "os.environ",
+        {
+            "DD_SERVICE": "test-service",
+            "DD_ENV": "test",
+            "DD_VERSION": "0.0.1",
+            "DD_TAGS": "team:testers",
```

Comment:
Doesn't look like there's any test coverage for multiple tags with `,` separators

Reply:
Thanks for the heads up, added it to one of the tests!